source/multiple-mongosyncs.txt

```diff
@@ -0,0 +1,255 @@
+.. _c2c-sharded-clusters:
+
+=====================================
+Use ``mongosync`` on Sharded Clusters
+=====================================
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+There are two ways to synchronize :ref:`sharded clusters
+<sharded-cluster>`. You can use either one ``monogosync`` or several
+``monogosync`` instances. For best performance, use multiple
+``monogosync`` instances, one ``monogosync`` for each shard in the
+cluster.
+
+.. _c2c-sharded-config-single:
+
+Configure a Single ``mongosync`` Instance
+-----------------------------------------
+
+To configure a single ``mongosync``, follow the :ref:`connection
+instructions <c2c-conn-top-level>` for your cluster architecture to
+connect to the :binary:`mongos` instance in your cluster.
+
+When you connect a single ``mongosync`` to a sharded cluster do not use
+the :urioption:`replicaSet` option or the :option:`id <mongosync --id>`
+option.
+
+The rest of this page addresses cluster to cluster synchronization
+using multiple ``monogosync`` instances.
+
+.. _c2c-sharded-config:
+
+Configure Multiple ``mongosync`` Instances
+------------------------------------------
+
+To configure multiple ``mongosync`` instances:
+
+#. :ref:`Verify cluster configuration <c2c-shard-config-verify>`
+#. :ref:`Determine the shard IDs <c2c-shard-config-determine>`
+#. :ref:`Connect the instances <c2c-shard-config-connect>`
+
+.. procedure::
+   :style: normal
+
+   .. step:: Verify cluster configuration
+
+      .. _c2c-shard-config-verify:
+
+      The source cluster and destination cluster must have the same
+      number of shards.
+
+   .. step:: Determine the shard IDs
+
+      .. _c2c-shard-config-determine:
+
+      To get the shard IDs, connect to the source cluster
+      :binary:`mongos` and run the :dbcommand:`listShards` command.
+
+      .. code-block:: javascript
+
+         db.adminCommand( { listShards: 1 } )
+
+      The information is in the ``shards`` array.
+
+      .. code-block:: javascript
+         :copyable: false
+         :emphasize-lines: 3,9
+
+           shards: [
+             {
+               _id: 'shard01',
+               host: 'shard01/localhost:27501,localhost:27502,localhost:27503',
+               state: 1,
+               topologyTime: Timestamp({ t: 1656612236, i: 2 })
+             },
+             {
+               _id: 'shard02',
+               host: 'shard02/localhost:27504,localhost:27505,localhost:27506',
+               state: 1,
+               topologyTime: Timestamp({ t: 1656612240, i: 4 })
+             }
+           ]
+
+   .. step:: 3. Connect the ``mongosync`` instances
```

Comment:
This step doesn't appear to show up in staging for me?

Reply:
The staging link was stale. It's updated now.